cpp/tests/sort/sort_test.cpp

```diff
@@ -197,6 +197,133 @@ TYPED_TEST(Sort, WithAllValid)
   }
 }
 
+TYPED_TEST(Sort, WithStructColumn)
```

Comment:
Can we add test cases for null ordering ? https://github.com/NVIDIA/spark-rapids/pull/1883#issuecomment-799087997

Reply:
Added with last commit.